cmd/merlinserver/main.go

```diff
@@ -41,9 +42,9 @@ import (
 	"github.com/satori/go.uuid"
 
 	// Merlin
-	"github.com/ne0nd0g/merlin/pkg"
```

Comment:
I believe best practice is to use the full repo URL. I've updated this upstream to fix the case sensitive error where ne0nd0g should have been Ne0nd0g. Please put back in the full URL path.